tests/ccl_test_distances.py

```diff
@@ -257,7 +308,24 @@ def compare_distances_mnu_hiz(z, chi_bench,dm_bench, Omega_v, w0, wa, Neff_mnu,
     dm = ccl.distance_modulus(cosmo,a[a_not_one])
 
     assert_allclose(dm, dm_bench[a_not_one], atol=1e-3, rtol = DISTANCES_TOLERANCE_MNU)
+
+@decorators.slow 
```

Comment:
We probably don't need to label this as slow - usually calling distances is pretty fast? We only want to use this label for things that take a really long time to compute and significantly slow down running the test script.